schemas/common.json

```diff
@@ -351,20 +351,28 @@
             "propertyOrder": 410,
             "advanced": true
         },
+        "vsd_ejabberd_license_file": {
```

Comment:
Please swap the order of `vsd_ejabberd_license_file` and `vsd_license_required_days_left` because the latter does not apply to the former. The days left only applies to the basic and cluster licenses, not the ejabberd license.

Reply:
Are you sure about the swap? I placed it here because all the ones above are license files. vsd_license_required_days_left would be sandwiched between license files.

Reply:
Yes, I am sure about the swap. Without the swap it looks like `vsd_license_required_days_left` applies to all licenses. It doesn't. I want `vsd_license_required_days_left` grouped only with the licenses it applies to. In fact, now that we have this 3rd license we should change the description for `vsd_license_required_days_left` to call out the licenses that are tested. Please make that change.

Reply:
Sounds good. Changed order and specified which licenses are tested.